File: longest_substring.py
# Be able to capture different string lengths that have no repeating characters
# Capture which string length is the greatest
# Output the longest string length

class Solution:
    def lengthOfLongestSubString(self,str):
        strings = [""]
        i = 0
        for letter in str:
            if strings[i].find(letter) == -1:
                strings[i].

File: longest_substring.js
function lengthOfLongestSubstring(str) {
  let strings = [];
  let max = 0;
  let index = 1;
  for (let letter of str) {
    strings.push(letter);
    for (let i = index; i < str.length; i++) {
      while (strings.includes(letter) !== true) {
        strings.push(str[i]);
        i++;
      }
      console.log(strings);
    }
    if (strings.length > max) {
      max = strings.length;
    }
    strings = [];
    index++;
  }
  return max;
}

console.log(lengthOfLongestSubstring("pwwkew"));
